audit-ci.json

```diff
@@ -8,6 +8,7 @@
     "GHSA-w5p7-h5w8-2hfq",
     "GHSA-3rfm-jhwj-7488",
     "GHSA-hhq3-ff78-jv3g",
-    "GHSA-76p3-8jx3-jpfq"
+    "GHSA-76p3-8jx3-jpfq",
+    "GHSA-hc6q-2mpp-qw7j"
```

Comment:
Can this vulnerability be fixed with `npm audit:fix`

Reply:
I didn't realise at the time but it gets fixed by this PR: https://github.com/mi6/ic-design-system/pull/341. Will remove this entry from the whitelist